splinterd/sample_configs/splinterd.toml.example

```diff
@@ -128,3 +128,32 @@ oauth_client_secret = "<client-secret>"
 
 # Redirect URL for the OAuth provider used by the REST API
 oauth_redirect_url = "http://localhost:8080/oauth/callback"
+
+
+#Appenders is the structure and stdout is the items name
+[appenders.stdout]
+#Type of appender options are stdout,stderr,file,rolling_file
+kind = "stdout"
+#Pattern controls the formatting of each log message.
+#Full reference `https://docs.rs/log4rs/1.0.0/log4rs/encode/pattern/index.html`
+pattern = "[ {d(%Y-%m-%d %H:%M:%S%.3f)}] T[{T}] {l} [{M}] {m}\n"
+
+[appenders.stderr]
+kind = "stderr"
+
+[appenders.rolling_file]
+kind = "rolling_file"
+#Filename is specific to rolling_file and file appenders and is simply a path to a file.
+filename = "/var/logs/my_file.log"
```

Comment:
/var/logs isn't a realistic path. FHS defines the location as /var/log. But more specifically, processes which are not run as root also need a subdirectory for permissions reasons. A realistic value here would be "/var/log/splinter/splinterd.log".